### ISSUE TITLE
You have validator issues

### ISSUE THREAD

**Issue by delg0030** [delg0030/portfolio#3]
Images should have an alt attribute, unless they have an ARIA role of "presentation" (2 results)
The web page should have a title that describes topic or purpose (1 result)
Text elements should have a reasonable contrast ratio (1 result)
